src/core/os/amdgpu/dri3/dri3WindowSystem.cpp

```diff
@@ -1177,20 +1254,7 @@ Result Dri3WindowSystem::AcquireScreenAccess(
 
     if (result == Result::Success)
     {
-        xcb_randr_get_output_info_cookie_t outputInfoCookie =
-            dri3Procs.pfnXcbRandrGetOutputInfo(pConnection, randrOutput, XCB_CURRENT_TIME);
-        xcb_randr_get_output_info_reply_t* pOutputInfoReply =
-            dri3Procs.pfnXcbRandrGetOutputInfoReply(pConnection, outputInfoCookie, NULL);
-
-        if (pOutputInfoReply)
-        {
-            randrCrtc = pOutputInfoReply->crtc;
-            free(pOutputInfoReply);
-        }
-        else
-        {
-            result = Result::ErrorInitializationFailed;
-        }
+        result = FindCrtcForOutput(hDisplay, pDevice, randrOutput, rootWindow, &randrCrtc);
```

Comment:
Should we skip this if result isn't Success (e.g., if pOutputInfoReply was null).

Reply:
OutputInfoReply is not needed anymore